src/main/java/org/folio/circulation/resources/handlers/error/CirculationErrorHandler.java

```diff
@@ -47,23 +43,6 @@
     return handleValidationError(error, errorType, succeeded(otherwise));
   }
 
-  public <T> Result<T> failWithValidationErrors(T otherwise) {
-    return failWithValidationErrors(succeeded(otherwise));
-  }
-
-  public <T> Result<T> failWithValidationErrors(Result<T> otherwise) {
-    List<ValidationError> validationErrors = errors.keySet().stream()
-      .filter(ValidationErrorFailure.class::isInstance)
-      .map(ValidationErrorFailure.class::cast)
-      .map(ValidationErrorFailure::getErrors)
-      .flatMap(Collection::stream)
-      .collect(toList());
-
-    return validationErrors.isEmpty()
-      ? otherwise
-      : failed(new ValidationErrorFailure(validationErrors));
-  }
```

Comment:
When we remove permission checking logic from the DeferFailureHandler we'll be able to reverse these changes